TrafficCapture/replayerPlugins/jsonMessageTransformers/Es2OsPost23/build.gradle

```diff
@@ -4,6 +4,9 @@ plugins {
 
 dependencies {
     implementation project(':replayerPlugins:jsonMessageTransformers:jsonMessageTransformerInterface')
+    implementation project(':replayerPlugins:jsonMessageTransformers:jsonJMESPathMessageTransformer')
+    implementation project(':replayerPlugins:jsonMessageTransformers:jsonJMESPathMessageTransformerProvider')
+    implementation group: 'io.burt', name: 'jmespath-core', version: '0.6.0'
```

Comment:
You shouldn't need these two.  Providers are basically just external entrypoints.  You're providing your own.  The jmespath library (or whatever implementation that the jsonJMESPathMessageTransformer package uses) will be picked automatically.
(see the next comment for how to shake the compile error that you would get)